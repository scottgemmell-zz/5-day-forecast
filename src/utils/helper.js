import moment from "moment";

export const toDayOfTheWeek = (val) => {
    return moment(val, "YYYY-MM-DD HH:mm:ss").format("dddd");
}

export const getTempStatus = (temp) => {
    let status = "";
    const num = parseFloat(temp);
    if(num > 8) {
        status = "hot";
    } else if(num > 4){
        status = "average";
    } else {
        status = "cold";
    }
    return status;
}