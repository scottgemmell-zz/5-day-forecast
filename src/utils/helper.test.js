import { getTempStatus } from "./helper";

describe("helper.js", () => {

	it("getTempStatus() ", () => {
		expect(getTempStatus(3)).toBe("cold");
		expect(getTempStatus(7)).toBe("average");
		expect(getTempStatus(11)).toBe("hot");
	});
});
