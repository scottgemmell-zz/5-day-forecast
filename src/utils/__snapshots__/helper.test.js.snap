// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`helper.js should do what I like 1`] = `"cold"`;
